contracts/WildcardsQV.sol

```diff
@@ -130,13 +130,11 @@ contract WildcardsQV is Initializable {
         returns (uint256 newProposalId)
     {
         proposalAddresses[proposalId] = _addressOfCharity;
-        //proposalOwner[proposalId] = msg.sender;
-        //benefactorsProposal[msg.sender] = proposalId;
         state[proposalId] = ProposalState.Active;
         emit LogProposalCreated(proposalId, _addressOfCharity);
         // So the first proposal will have an ID of 0
         proposalId = proposalId.add(1);
-        return proposalId;
+        return proposalId.sub(1); // <- so it is returning the ID of the created proposal
```

Comment:
Good idea. Yeah proposal 0 shouldn't get an unfair advantage. The reason it was starting from 0 so was that it would map the exactly to the benefactorId in the wildcards contracts which someone mentioned started at 0